.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v3
         with:
-          dotnet-version: 3.1.x
+          dotnet-version: 6.0.x
```

Comment:
@RushabhBhansali please remove all these versions.

```yml
with:
  dotnet-version: 6.0.x
```

`actions/setup-dotnet` will read the `global.json` file to get the SDK version. Having the version defined here is just duplication.